portal-impl/src/com/liferay/portlet/messageboards/service/impl/MBMessageLocalServiceImpl.java

```diff
@@ -1556,9 +1556,9 @@ public MBMessage updateMessage(
 
 			thread.setPriority(priority);
 
-			mbThreadPersistence.update(thread);
-
 			updatePriorities(thread.getThreadId(), priority);
+
```

Comment:
Hey @hudakl we should undo this change and keep the original order. First update persistence and then update priorities, since the fix for the bug is not related with reordering this.